crates/ruff_linter/src/rules/flake8_todos/snapshots/ruff_linter__rules__flake8_todos__tests__missing-todo-link_TD003.py.snap

```diff
@@ -39,12 +39,14 @@ TD003.py:29:9: TD003 Missing issue link on the line following this TODO
 31 | # TODO: here's a TODO on the last line with no link
    |
 
-TD003.py:31:3: TD003 Missing issue link on the line following this TODO
+TD003.py:31:3: TD003 Missing issue link for this TODO
    |
 29 | # foo # TODO: no link!
 30 | 
 31 | # TODO: here's a TODO on the last line with no link
    |   ^^^^ TD003
+32 | 
+33 | # TODO: https://github.com/astral-sh/ruff/issues/8061 This comment also has a link
```

Comment:
Can you add a test case for `#002` variant as well?

Reply:
Yep! It's at the bottom of TD003.py, but doesn't error, so is not in the `.snap`. The link-case is included due to being within the context of the non-linnk failing example :-)